gtsfm/bundle/bundle_adjustment.py

```diff
@@ -62,7 +72,13 @@ def run(
             return initial_data
 
         # noise model for measurements -- one pixel in u and v
-        measurement_noise = gtsam.noiseModel.Isotropic.Sigma(IMG_MEASUREMENT_DIM, 1.0)
+        if self._robust_measurement_noise:
+            measurement_noise = gtsam.noiseModel.Robust(
+                gtsam.noiseModel.mEstimator.Huber(1.35),
+                gtsam.noiseModel.Isotropic.Sigma(IMG_MEASUREMENT_DIM, 1.0),
+            )
```

Comment:
is this the correct way to do it? I am using the same object of noise model in all factors. Should I create a copy every time? @ProfFan

Reply:
I don't think you need to do that